src/mds/MDSRank.cc

```diff
@@ -3066,11 +3066,21 @@ bool MDSRank::evict_client(int64_t session_id,
     return false;
   }
 
+  auto& addr = session->info.inst.addr;
+  {
+    std::stringstream _ss;
+    auto& ss = _ss.get_stream();
+    ss << "Evicting " << (blacklist ? "(and blacklisting) " : "")
+       << "client session " << session_id << " (" << addr << ")";
+    dout(1) << ss.strv() << dendl;
+    clog->info() << ss.strv();
```

Comment:
I'm not well-versed in `std::stringstream`, but it doesn't look too complicated. Something like this:

```
{
  std::stringstream ss;
  ss << "Evicting " << (blacklist ? "(and blacklisting) " : "")
       << "client session " << session_id << " (" << addr << ")";
  dout(1) << ss.str() << dendl;
  clog->info() << ss.str();
}
```

@batrick Does that look right?

Reply:
Yes!

Reply:
@smithfarm @batrick thank you for the help. I have made the changes and commit.